src/modules/wali/x86-64-linux/X86_64LinuxWaliModule.v3

```diff
@@ -109,6 +135,98 @@ class WaliInstance {
 	def exit_group(args: Range<Value>) -> HostResult {
 		return HostResult.Throw(Exit.new(Values.unbox_u(args[0])));
 	}
+	def sched_getaffinity(args: Range<Value>) -> HostResult {
+		return HostResult.Throw(Exit.new(Values.unbox_u(args[0])));
+	}
+	def set_tid_address(args: Range<Value>) -> HostResult {
+		var range = getRegionOf(args, 0, 4);
+		if (range.reason != TrapReason.NONE) return range.toHostResultThrow();
+		return syscall(LinuxConst.SYS_set_tid_address, 
+			(Pointer.atContents(range.result))
+		);
+	}
+	def mmap(args: Range<Value>) -> HostResult {
+		var range = getRegion(args, 0);
+		if (range.reason != TrapReason.NONE) return range.toHostResultThrow();
+		var sysfd = getSysFd(args, 4);
+		if (sysfd < 0) return EBADFD;
+		var prot = Values.unbox_l(args[2]);
+		var flags = Values.unbox_l(args[3]);
+		var offset = Values.unbox_l(args[5]);
+		return syscall(LinuxConst.SYS_mmap, 
+			(Pointer.atContents(range.result), range.result.length, prot, flags, sysfd, offset)
+		);
+	}
+	def fcntl(args: Range<Value>) -> HostResult {
+		return HostResult.Throw(Exit.new(1));
```

Comment:
As per discussion today, this should be a Trap with a `TrapReason.UNIMPLEMENTED`.